libnopegl/src/api.c

```diff
@@ -195,6 +197,17 @@ static int backend_init(struct ngl_backend *backend, struct gpu_ctx *gpu_ctx)
     return 0;
 }
 
+static int backend_copy(struct ngl_backend *dst, const struct ngl_backend *src)
+{
+    *dst = *src;
+
+    dst->caps = ngli_memdup(dst->caps, dst->nb_caps * sizeof(*dst->caps));
```

Comment:
nit: can you use `src` in the `ngli_memdup` arguments?

Reply:
Done.